README.md

```diff
@@ -12,9 +12,14 @@ For more information on farmOS, visit [farmOS.org](https://farmOS.org).
 
 ## Usage
 
-1. Create an HTML element with an ID, eg: `<div id="farm-map"></div>`
-2. Call the map creation method with the element ID: `farmOS.map.create('farm-map');`
-3. (optional) Add behaviors - see below.
+1. Include `farmOS-map.js` and `farmOS-map.css` in the page:
+  ```html
+<link rel="stylesheet" href="./farmOS-map.css" type="text/css">
```

Comment:
Need to figure out where the OpenLayers CSS

Reply:
I've double-checked and the contents of `ol.css` is in fact getting bundled into `farmOS-map.css` after these changes - I just got thrown off on the call because it really isn't a huge amount of CSS so the file was smaller than I naively expected.